src/__tests__/verify.v3.e2e.test.ts

```diff
@@ -0,0 +1,36 @@
+import { handler } from "../commands/verify";
```

Comment:
Weird to have this file name if the other one is not labelled v2.

Reply:
Changed verify.e2e.test.ts to verify.v2.e2e.test.ts